e2e/_suites/fleet/fleet.go

```diff
@@ -384,45 +362,70 @@ func (fts *FleetTestSuite) installCerts() error {
 	if err != nil {
 		log.WithFields(log.Fields{
 			"agentVersion":      common.ElasticAgentVersion,
-			"agentStaleVersion": common.AgentStaleVersion,
+			"agentStaleVersion": fts.Version,
 			"error":             err,
 			"installer":         agentInstaller,
-			"version":           fts.Version,
 		}).Error("Could not install the certificates")
 		return err
 	}
 
+	log.WithFields(log.Fields{
+		"agentVersion":      common.ElasticAgentVersion,
+		"agentStaleVersion": fts.Version,
+		"error":             err,
+		"installer":         agentInstaller,
+	}).Tracef("Certs were installed")
 	return nil
 }
 
-func (fts *FleetTestSuite) anAgentIsUpgraded(desiredVersion string) error {
+func (fts *FleetTestSuite) anAgentIsUpgradedToVersion(desiredVersion string) error {
 	switch desiredVersion {
-	case "stale":
-		desiredVersion = common.AgentStaleVersion
 	case "latest":
 		desiredVersion = common.ElasticAgentVersion
-	default:
-		desiredVersion = common.ElasticAgentVersion
 	}
+	log.Tracef("Desired version is %s. Current version: %s", desiredVersion, fts.Version)
 
 	agentService := deploy.NewServiceRequest(common.ElasticAgentServiceName)
+
+	/*
```

Comment:
Keeping the upgrade-using-command code in a comment, but I will remove it before merging the PR